jboss-eap-standalone-rhel8/README.md

```diff
@@ -41,38 +40,38 @@ This template creates all of the compute resources to run JBoss EAP 7 on top of
 Following is the Architecture :
 <img src="images/RHEL8-Arch.PNG" width="800">
 
-To learn more about JBoss Enterprise Application Platform, check out:
+To learn more about JBoss Enterprise Application Platform, visit:
 https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/
 
-## Licenses and Costs
+## Licenses, Subscriptions and Costs
 
-This uses RHEL 8.0 image which is a PAY AS YOU GO image and doesn't require the user to license it, it will be licensed automatically after the instance is launched first time and user will be charged hourly in addition to Microsoft's Linux VM rates.  Click [here](https://azure.microsoft.com/en-gb/pricing/details/virtual-machines/linux/#red-hat) for pricing details. You also need to have a RedHat account to register to Red Hat Subscription Manager (RHSM). Click [here](https://access.redhat.com/products/red-hat-subscription-management) to know more about RHSM and pricing.
+This template uses a RHEL 8.0 image which is a Pay-as-you-go (PAYG) image and does not require the user to license it.  The VM will be licensed automatically after the instance is launched for the first time and the user will be charged hourly in addition to Microsoft's Linux VM rates.  Click [here](https://azure.microsoft.com/en-gb/pricing/details/virtual-machines/linux/#red-hat) for pricing details. You will also need to have a RedHat Subscription.  Click [here](https://access.redhat.com/products/red-hat-subscription-management) to learn more about Red Hat Subscription Manager (RHSM), pricing and registeration for RHSM.
 
 ## Prerequisites
 
-1. Azure Subscription with specified payment method (RHEL 8 is a market place product and requires payment method to be specified in Azure Subscription)
+1. Azure Subscription with specified payment method (RHEL 8 is a marketplace product and requires a payment method to be specified in the Azure Subscription)
 
 2. To deploy the template, you will need to:
 
-   - Choose an admin user name and password for your VM.
+   - **Admin Username** and password.
     
-   - Choose a name for your VM.
+   - **VM Name** 
```

Comment:
Modified this line as follows. Please suggest

DNS Label Prefix for the public IP which should be unique. Note that this will also be the name of your VM.